### ISSUE TITLE
Compile error after xcode Version 12.0.1 update on BoundingBox.swift

### ISSUE THREAD

**Issue by kutlueren** [SwiftyTesseract/SwiftyTesseract#77]
Hi,

I have recently updated xcode for ios 14 development and then I get the following error on build. I have tried to compile it by Swift version both 4.2 and 4 but didn't work.

Could you check it out when you have time? thanks

XCode version 12.0.1 
Swift Version 5

![Screen Shot 2020-09-29 at 10 08 12](https://user-images.githubusercontent.com/4067167/94524763-ba7cd000-023b-11eb-8094-ae9aac536c79.png)

**Comment by kutlueren** [comment]
I think this seems to be the solution;

import Foundation

struct BoundingBox {
  var originX: Int32 = 0
  var originY: Int32 = 0
  var widthOffset: Int32 = 0
  var heightOffset: Int32 = 0

```
  var cgRect: CGRect {
    return CGRect(
        x: Int(originX),
        y: Int(originY),
        width: Int((widthOffset - originX)),
        height: Int((heightOffset - originY))
    )
  }
}
```

**Comment by Steven0351** [comment]
I'm working on getting a fix for this pushed out. I had anticipated I would be able to have version 4.0.0 of the library out (the develop branch currently has a fix for this), but unfortunately there are issues with SPM and testing libraries that contain Swift Package Resources. Since I can't release that in a timely manner, I should be releasing a version 3.1.3 soon barring any issues with Xcode 12 with the 3.x.x version.

**Comment by Steven0351** [comment]
Just an FYI, I'm having issues with getting this to run in the old configuration with iOS 14.0 for the simulator. It mentions that it;s building for iOS Simulator ARM instead of x86_64, which is a head scratcher for me because I'm definitely not on Apple Silicon so I'm not sure why the simulator architecture would be ARM.

**Comment by juchampagne** [comment]
@Steven0351 you need to exclude arm64 for simulator builds in the project settings:

<img width="606" alt="Screenshot 2020-10-05 at 11 52 03" src="https://user-images.githubusercontent.com/17700424/95071262-4a6dbe80-0701-11eb-892b-e3305996f90d.png">

Hope this helps.

**Comment by Steven0351** [comment]
This should now be resolved with version 3.1.3

**Comment by Mattijah** [comment]
Hi @Steven0351, thanks for the fix. I'm trying to update SwiftyTesseract from `3.1.2` to `3.1.3`, but unfortunately I'm getting the following error. Any idea whether this relates to the new Xcode (I'm still on 11.6) or perhaps different cocoapods version...?

![Screenshot 2020-10-18 at 22 45 47](https://user-images.githubusercontent.com/3887042/96385371-b5a0a180-1193-11eb-8962-61209da05101.png)

![Screenshot 2020-10-18 at 22 49 44](https://user-images.githubusercontent.com/3887042/96385498-9a826180-1194-11eb-8434-566dd6d1370b.png)

**Comment by Steven0351** [comment]
Can your first attempt with Xcode 12 and if that doesn’t work then attempt with Cocoapods 1.10? The readme in the support/3.x.x branch gives some reasoning behind needing that Cocoapods version.

**Comment by Mattijah** [comment]
I edited my comment saying that it works ok when running on a real device, so the problem will most likely be somewhere else. I will download the new Xcode and try it though....

**Comment by Steven0351** [comment]
Yeah, I believe that issue is related to Cocoapods itself. I had the same issue when trying to lint the podspec; the problem lies with it trying to build for arm64 with the simulator

**Comment by Mattijah** [comment]
So I've installed the same version of pods `1.10.0.rc.1` and now I'm getting this error:

![Screenshot 2020-10-24 at 19 57 32](https://user-images.githubusercontent.com/3887042/97088811-b6be4c80-1633-11eb-99b5-8cfaba44d1e9.png)

**Comment by Steven0351** [comment]
> 
> // Compiles on Xcode 12.
So the error is with CocoaPods 1.10.rc.1 & Xcode 12?

**Comment by Mattijah** [comment]
@Steven0351 Correct. After using the new CocoaPods I got a different error and then when using Xcode 12 it all compiled ok. I had to however change the podspec of my pod too as I wasn't able to push it due to this error: "_error: could not find module 'SwiftyTesseract' for target 'arm64-apple-ios-simulator'; found: x86_64-apple-ios-simulator, x86_64_". It sucks a bit all of this, but it is what it is... 🤷‍♂️
